packages/patternfly-4/react-core/src/components/Tabs/Tabs.js

```diff
@@ -49,8 +50,34 @@ class Tabs extends React.Component {
   id = getUniqueId();
   tabList = React.createRef();
 
-  handleTabClick(event, eventKey) {
+  handleTabClick(event, eventKey, tabChildId, tabChildRef) {
     this.props.onSelect(event, eventKey);
+    // process any children outside of the component
+    if (tabChildRef) {
+      React.Children.map(this.props.children, (child, i) => {
+        const section = ReactDOM.findDOMNode(child.props.tabChildRef.current);
```

Comment:
You may not be passing in the reference properly. You should be able to access the tab child object directly without searching the dom.

Check out this approach https://reactjs.org/docs/forwarding-refs.html

Reply:
Yup, that's what I was doing earlier and it causes warnings for the docs.

Reply:
I believe that's a Type issue we need to resolve. 

If the components property is expecting a certain type, it's possible TabChild.d.ts is not defined properly? Or, our docs generator needs to be more flexible -- I believe @jschuler worked on that?

As a test, you can try something like this below. If it works, then it's probably a Type issue.

```
components: {
    Tabs,
    Tab
    Tab,
    TabChild as any
  },
```